core/src/main/java/org/arquillian/cube/impl/reporter/TakeCubeInformation.java

```diff
@@ -0,0 +1,43 @@
+package org.arquillian.cube.impl.reporter;
+
+import org.arquillian.cube.spi.Cube;
+import org.arquillian.cube.spi.CubeRegistry;
+import org.arquillian.cube.spi.event.lifecycle.AfterAutoStop;
+import org.arquillian.cube.spi.metadata.CanReportMetrics;
+import org.arquillian.recorder.reporter.PropertyEntry;
+import org.arquillian.recorder.reporter.Reportable;
+import org.arquillian.recorder.reporter.event.PropertyReportEvent;
+import org.jboss.arquillian.core.api.Event;
+import org.jboss.arquillian.core.api.annotation.Inject;
+import org.jboss.arquillian.core.api.annotation.Observes;
+
+import java.util.List;
+
+/**
+ * Observer that is executed before suite and prints all metrics reported by registered cubes.
+ */
+public class TakeCubeInformation {
```

Comment:
Basically I followed the same convention used in Recorder integrations. They use this Take + <blabla>. We can change it but this is the reason why I used this name